src/js/model/primitives/Primitive.js

```diff
@@ -24,18 +30,48 @@ function clean(spec, resolve) {
       delete spec[k];
     }
     else if (dl.isObject(p)) {
-      spec[k] = p.signal && res ? sg.value(p.signal) : clean(spec[k], resolve);
+      spec[k] = p.signal && cln ? sg.value(p.signal) : _clean(spec[k], clean);
     }
   }
 
   return spec;
 }
 
-prototype.export = function(resolve) {
-  return clean(dl.duplicate(this), resolve);
+/**
+ * Initializes the Primitive. This is often where properties will be replaced
+ * with Lyra-specific signals to drive reactive updates.
+ * @return {Object} The Primitive.
+ */
+Primitive.prototype.init = function() {
+  return this;
 };
 
-prototype.parent = function(pid) {
+/**
+ * Exports the primitive as a complete Vega specification.
+ * @param  {boolean} [clean=true] - Should Lyra-specific properties be removed
+ * or resolved (e.g., converting property signal references to actual values).
+ * @return {Object} A Vega specification.
+ */
+Primitive.prototype.export = function(clean) {
+  return _clean(dl.duplicate(this), clean);
+};
+
+/**
+ * Exports the primitive as a complete Vega specification with extra definitions
+ * to power Lyra-specific interaction (e.g., extra manipulator mark definitions).
+ * @return {Object} A Vega specification.
```

Comment:
I believe that's a related but separate issue -- `@return {this}` would be useful for chaining, but that's actually a step beyond what I was advocating for / wondering why it didn't seem to be "supported," per @deathbearbrown. If I see `@returns {Object}` I would expect it to be either an object literal or a `new Object()`. If it returns an instance of a constructor, with all the functionality that being an instance implies, I am used to (from YUIdoc) `@returns {MyClass}`. If JSDoc really provides no facility to specify a custom constructor as its return type I am both disappointed and surprised.